COMDOX-1734: Updated SCP topics for clarity (#941)
commerce-docs/microsite-commerce-storefront: src/content/docs/releases/index.mdx

Store administrators can place and manage orders on behalf of customers. Two new User Account containers — [`SellerAssistedBuyingActivity`](/dropins/user-account/containers/seller-assisted-buying-activity/) and [`SellerAssistedBuyingSettings`](/dropins/user-account/containers/seller-assisted-buying-settings/) — let customers view their assisted-buying history and control admin access. A header banner appears when a representative is active. See [Seller-assisted buying](/dropins/user-account/containers/seller-assisted-buying/) for setup details ([#1283](https://github.com/hlxsites/aem-boilerplate-commerce/pull/1283)).

#### Adobe Commerce 2.4.9 support (Commerce on Cloud or on-premises)

> SCP-4.9+ is now supported for Commerce on Cloud or on-premises with Adobe Commerce Optimizer. Adobe Commerce as a Cloud Service is not yet synchronized with SCP-4.9 — those environments remain on Adobe Commerce 2.4.8-p5. See [Adobe Commerce 2.4.9](/setup/configuration/storefront-compatibility/v249/) for Storefront Compatibility Package details.

Reviewer: ```suggestion
SCP-4.9+ is now supported for Commerce on Cloud or on-premises with Adobe Commerce Optimizer. Adobe Commerce as a Cloud Service is not yet synchronized with SCP-4.9. Those environments remain on Adobe Commerce 2.4.8-p5. See [Adobe Commerce 2.4.9](/setup/configuration/storefront-compatibility/v249/) for Storefront Compatibility Package details.
```

> Adobe Commerce 2.4.9 is now supported for Commerce on Cloud or on-premises without Adobe Commerce Optimizer. Adobe Commerce as a Cloud Service, Adobe Commerce Optimizer, and Commerce on Cloud or on-premises with Adobe Commerce Optimizer are not yet synchronized with 2.4.9 — those environments remain on Adobe Commerce 2.4.8-p5. See [Adobe Commerce 2.4.9](/setup/configuration/storefront-compatibility/v249/) for Storefront Compatibility Package details.

Reviewer 2: ```suggestion
SCP-4.9+ is now supported for Commerce on Cloud or on-premises with Adobe Commerce Optimizer. Adobe Commerce as a Cloud Service is not yet synchronized with SCP-4.9 — those environments remain on Adobe Commerce 2.4.8-p5. See [Adobe Commerce 2.4.9](/setup/configuration/storefront-compatibility/v249/) for Storefront Compatibility Package details.
```

#### Adobe Commerce 2.4.9 support

> Adobe Commerce 2.4.9 is now supported on Adobe Commerce as a Cloud Service, Adobe Commerce Optimizer, and Commerce on Cloud or on-premises with Adobe Commerce Optimizer. See [Adobe Commerce 2.4.9](/setup/configuration/storefront-compatibility/v249/) for Storefront Compatibility Package details.

Reviewer 2: Adobe Commerce 2.4.9 is a separate offering and does not support Adobe Commerce as a Cloud Service (ACCS), Adobe Commerce Optimizer (ACO), or Commerce on Cloud/on-premises with Adobe Commerce Optimizer.

Also, the Adobe Commerce as a Cloud Service codebase has not yet been synchronized with the latest Adobe Commerce 2.4.9 release. It is currently synced with Adobe Commerce 2.4.8-p5. 

Ref. -
https://wiki.corp.adobe.com/spaces/RM/pages/3740660771/2026+May+Release+Summary+2.4.9+2.4.8-p5+2.4.7-p10+2.4.6-p15+2.4.5-p17+2.4.4-p18

|----------------|----------------------------------|
| `2.4.7` | `4.7.13` |
| `2.4.8` | `4.8.25` |
| `2.4.9` | `4.9.0` (see [who installs](/setup/configuration/storefront-compatibility/#find-your-path) — automatic on Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer. Manual on Commerce on Cloud or on-premises with Adobe Commerce Optimizer) |

| Adobe Commerce B2B | Storefront Compatibility B2B Package |
|--------------------|--------------------------------------|

> | `1.5.2` | `1.0.22` (see [who installs](/setup/configuration/storefront-compatibility/#storefront-compatibility-b2b-package) — automatic on Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer. Manual on Commerce on Cloud or on-premises with Adobe Commerce Optimizer) |

Reviewer 2: It's unclear to me how SCP-B2B can be installed in an Adobe Commerce Optimizer-only environment.
